src/corehost/common/pal.unix.cpp

```diff
@@ -79,12 +79,7 @@ bool pal::load_library(const string_t* path, dll_t* dll)
 
 pal::proc_t pal::get_symbol(dll_t library, const char* name)
 {
-    auto result = dlsym(library, name);
-    if (result == nullptr)
-    {
-        trace::error(_X("Failed to resolve library symbol %s, error: %s"), name, dlerror());
```

Comment:
would it make sense to at least `trace::log` this?  Or whatever the 'verbose' logging is here.  That way it still is debuggable with COREHOST_TRACE.

Reply:
Yes. Added logging to both Windows and Linux\OSX.

The logging here is not necessary since each error cases does their own logging, although that logging is not as precise as doing it here, plus as you mentioned it may help debugging (esp non-error cases), so I added it.